From 19e65e204b5456170c63f9b59595bcc2cb2ff983 Mon Sep 17 00:00:00 2001
From: Stephen L Arnold <nerdboy@gentoo.org>
Date: Wed, 25 Aug 2021 17:08:09 -0700
Subject: [PATCH] patch network tests for CI/packaging

---
 test.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/test.c b/test.c
index f830695..1fff29f 100644
--- a/test.c
+++ b/test.c
@@ -788,6 +788,7 @@ static void test_blocking_connection_errors(void) {
     struct addrinfo *ai_tmp = NULL;
 
     int rv = getaddrinfo(HIREDIS_BAD_DOMAIN, "6379", &hints, &ai_tmp);
+    #if 0
     if (rv != 0) {
         // Address does *not* exist
         test("Returns error when host cannot be resolved: ");
@@ -809,6 +810,7 @@ static void test_blocking_connection_errors(void) {
         printf("Skipping NXDOMAIN test. Found evil ISP!\n");
         freeaddrinfo(ai_tmp);
     }
+    #endif
 
 #ifndef _WIN32
     test("Returns error when the port is not open: ");
@@ -817,10 +819,12 @@ static void test_blocking_connection_errors(void) {
         strcmp(c->errstr,"Connection refused") == 0);
     redisFree(c);
 
+    #if 0
     test("Returns error when the unix_sock socket path doesn't accept connections: ");
     c = redisConnectUnix((char*)"/tmp/idontexist.sock");
     test_cond(c->err == REDIS_ERR_IO); /* Don't care about the message... */
     redisFree(c);
+    #endif
 #endif
 }
 
@@ -1486,6 +1490,7 @@ int main(int argc, char **argv) {
     test_blocking_connection_errors();
     test_free_null();
 
+    #if 0
     printf("\nTesting against TCP connection (%s:%d):\n", cfg.tcp.host, cfg.tcp.port);
     cfg.type = CONN_TCP;
     test_blocking_connection(cfg);
@@ -1494,6 +1499,7 @@ int main(int argc, char **argv) {
     test_invalid_timeout_errors(cfg);
     test_append_formatted_commands(cfg);
     if (throughput) test_throughput(cfg);
+    #endif
 
     printf("\nTesting against Unix socket connection (%s): ", cfg.unix_sock.path);
     if (test_unix_socket) {
-- 
2.32.0

